kpi/management/commands/remove_s3_orphans.py

```diff
@@ -0,0 +1,154 @@
+#!/usr/bin/env python
+# vim: ai ts=4 sts=4 et sw=4 coding=utf-8
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+import codecs
+import re
+import sys
+import time
+
+from django.conf import settings
+from django.core.management.base import BaseCommand, CommandError
+from django.core.files.storage import get_storage_class
+from django.utils.translation import ugettext as _
+
+from kpi.models.import_export_task import ExportTask
+
+
+# S3 Monkey Patch
+from boto import handler
+from boto.resultset import ResultSet
+from boto.s3.bucket import Bucket
+import xml.sax
+import xml.sax.saxutils
+
+
+def _get_all(self, element_map, initial_query_string='',
```

Comment:
From comparing this to the original boto `_get_all()`, I think the point of the monkey patch is to deal with bad characters that are returned by S3. Is that right? If so, let's put a docstring on this function explaining why it needs to be here.